server/testutil/app/app.go

```diff
@@ -54,7 +54,7 @@ func RunWithApp(t *testing.T, app *App, commandPath string, commandArgs []string
 		"--static_directory=static",
 		"--app_directory=/app",
 		fmt.Sprintf("--app.build_buddy_url=http://localhost:%d", app.HttpPort),
-		"--database.data_source=sqlite3://:memory:",
```

Comment:
Any reason you don't want to use the workaround suggested in the linked thread?

Reply:
The workaround involves 3 different DB flags, which seemed less clean. I don't feel particularly strongly one way or the other but in my experience the on-disk DB is very fast so I figured this would be OK.

Reply:
sgtm